Amend the style guide for line wrapping (#43279)
kubernetes/website: content/en/docs/contribute/style/style-guide.md

any preceding paragraphs or texts. A two line spacing helps visualize the overall
structure of content in a code editor better.

> Manually wrap a paragraph when appropriate. Since the git tool and the github website

Reviewer: ```suggestion
Manually wrap paragraphs in the Markdown source when appropriate.
Since the git tool and the GitHub website
```

> generates file diffs on a line-by-line basis, manually wrapping long lines helps

Reviewer: ```suggestion
generate file diffs on a line-by-line basis, manually wrapping long lines helps
```

> the reviwers to easily find out the changes made in a PR and provide feedbacks.

Reviewer 2: ```suggestion
the reviewers to easily find out the changes made in a PR and provide feedbacks.
```

> the reviewers to easily find out the changes made in a PR and provide feedbacks.

Reviewer 3: ```suggestion
the reviewers to easily find out the changes made in a PR and provide feedback.
```

> It aslo helps the downstream localization teams where people track the upstream

Reviewer 3: ```suggestion
It also helps the downstream localization teams where people track the upstream
```

changes on a per-line basis. Line wrapping can happen at the end of a sentence

> or a punctuation character, for example. One exception to this is that a markdown

Reviewer: ```suggestion
or a punctuation character, for example. One exception to this is that a Markdown
```

any preceding paragraphs or texts. A two line spacing helps visualize the overall
structure of content in a code editor better.

Manually wrap paragraphs in the Markdown source when appropriate. Since the git

> tool and the github website generate file diffs on a line-by-line basis,

Reviewer: ```suggestion
tool and the GitHub website generate file diffs on a line-by-line basis,
```
